ietf/group/views.py

```diff
@@ -771,13 +780,12 @@ def dependencies(request, acronym, group_type=None):
     graph = {
         "nodes": [
             {
-                "id": x.name,
-                "rfc": x.get_state("draft").slug == "rfc",
-                "post-wg": not x.get_state("draft-iesg").slug
-                in ["idexists", "watching", "dead"],
-                "expired": x.get_state("draft").slug == "expired",
-                "replaced": x.get_state("draft").slug == "repl",
-                "group": x.group.acronym if x.group.acronym != "none" else "",
+                "id": x.became_rfc().name if x.became_rfc() else x.name,
```

Comment:
Check this on a busy group and if the time is noticeable, I think we can get the db to grab this with the primary query instead of n+1 -ing it here.

Reply:
We might do that in a way the puts the conditional into the query as well, so that we don't see this pattern repeating.

Reply:
On the idr group (see below to gauge whether this is "busy", but it looks it to me), this brute force approach takes the request from 2-2.4 seconds up to 2.2-2.5 seconds on my dev machine. The code here is taking advantage of the caching in `became_rfc()`, so I believe it's at least only paying the N+1 penalty and not hammering the db any worse than that.

<img width="401" alt="image" src="https://github.com/ietf-tools/datatracker/assets/19472766/2c34c700-296d-4d77-a7a2-f28ddc891d02">